### ISSUE TITLE
Unable to create groups using sentry natively on php

### ISSUE THREAD

**Issue by anoopmd** [cartalyst/sentry#502]
I am using Sentry v2.1.4.
I using it natively on php through composer.

I am getting an issue while creating groups.

This is the error log
SQLSTATE[42S02]: Base table or view not found: 1146 Table 'test.groups' doesn't exist (SQL: select * from `groups` where `name` = Moderator limit 1)

{My code}
$group = Sentry::createGroup(array(
        'name'        => 'Moderator',
        'permissions' => array(
            'admin' => 1,
            'users' => 1,
        ),
      ));
{/My code}

Can somebody please me help me ?

**Comment by stefanmajoor** [comment]
Well, I think that the error message speaks for itself: 

``` 
 Table 'test.groups' doesn't exist
``` 

This tables does not exist in the database, probably because the necessary migrations were not run.

**Comment by anoopmd** [comment]
@stefanmajoor. Thanks for the response.

I solved it myself.
I knew from the error message that the tables were msiing and the necessary migrations were not run.

The official documentation at https://cartalyst.com/manual/sentry/2.1 does not mention how to perform migration when using sentry natively on php. 

After digging the internet for close to three hours, I found the solution of how to run the migration
from the following links
http://www.ibm.com/developerworks/library/se-sentry/index.html
http://www.ibm.com/developerworks/library/se-sentry2/
